source/conf.py.BAK

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
This should be removed.

Reply:
If you want to keep for historical purpose, add it in master first then overwrite it in this branch. Then we can see what has been changes.